stable/aws-for-fluent-bit/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 name: aws-for-fluent-bit
 description: A Helm chart to deploy aws-for-fluent-bit project
 version: 0.1.33
```

Comment:
do you need to bump the chart version so we can have a new release for this?

Reply:
@PettitWesley Sure I will update the chart version - It's a good idea. 
My impression was that it seems to be a minor version upgrade but I agree with your thoughts.